lib/worker.js

```diff
@@ -58,14 +60,8 @@ function shim(Linter) {
   return LinterShim;
 }
 
-function loadDefaultTSLint() {
-  if (!tslintDef) {
-    // eslint-disable-next-line import/no-dynamic-require
-    tslintDef = require(tslintModuleName).Linter;
-  }
-}
-
-async function getLocalLinter(basedir) {
+function resolveAndCacheLinter(fileDir, moduleDir) {
+  const basedir = moduleDir || fileDir;
```

Comment:
🐛 `baseDir` please 😉

Reply:
I wasn't sure how you wanted me to deal with that, since it previously *was* called `basedir`, apparently for the sake of brevity on line 71. I guess changing that line to `{ basedir: baseDir }` wouldn't be the end of the world.